eng/pipeline/stages/pool-core.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+# This template defines common pool names for dnceng agents for various platforms.
+
+parameters:
+  - name: public
+    type: boolean
+  - name: servicing
+    type: boolean
+  # The inner template: { template string, parameters object }
+  - name: inner
+    type: object
+
+stages:
+  - template: ${{ parameters.inner.template }}
+    parameters:
+      ${{ insert }}: ${{ parameters.inner.parameters }}
+
+      dncengPool:
+        # Pick pool. https://github.com/dotnet/arcade/blob/0db07252ccb18afdf94820ba6125da6de729ec04/Documentation/AzureDevOps/AzureDevOpsOnboarding.md#agent-queues
+        ${{ if parameters.public }}:
+          ${{ if parameters.servicing }}:
+            name: NetCore1ESPool-Svc-Public
+          ${{ if not(parameters.servicing) }}:
```

Comment:
FYI you can do `{{ else }}` now in .yml: https://docs.microsoft.com/en-us/azure/devops/release-notes/2021/sprint-192-update#new-yaml-conditional-expressions